src/snowcli/cli/sql/commands.py

```diff
@@ -8,7 +8,11 @@
 from snowcli.output.decorators import with_output
 from snowcli.output.types import QueryResult, CommandResult, MultipleResults
 
+# simple Typer with defaults because it won't become a command group as it contains only one command
```

Comment:
I don't have a better idea at the moment. In the future we can think about an util function which will be responsible for creating a typer of specific type (for example: "typer for command group with only one command").

Reply:
More comments about it: https://github.com/Snowflake-Labs/snowcli/pull/385#discussion_r1328888405